app/utils/sites.py

```diff
@@ -12,7 +12,7 @@
 
 
 def get_token(api_url: str, login: str, pwd: str) -> str:
-    response = requests.post(f"{api_url}/login/access-token", data={"username": login, "password": pwd}, timeout=3)
+    response = requests.post(f"{api_url}/login/access-token", data={"username": login, "password": pwd}, timeout=10)
```

Comment:
The API login was timing out on me every other time with only a 3 second wait (perhaps because I am in the U.S.?).

Reply:
it might be the case I have difficulties in Chile too, we can update this but please do it in an other PR it's not the topic here :)